Tools/TestWebKitAPI/Tests/WebKit/backgroundcolor.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<meta name='viewport' content='width=device-width, initial-scale=1'>
+<body>
+<div>
+    hello
+  <div style="width: 100px; height: 100px; background-color: blue">
```

Comment:
Do we need a new test page, or can you use the existing `red.html`?

Reply:
well that's a very aptly named and useful existing file lol, I shall use that